### ISSUE TITLE
Error when loading the model

### ISSUE THREAD

**Issue by skyimager** [huster-wgm/geoseg#7]
When loading the model using the below code:

```python
checkpoint_file = "BR-Net_iter_5000.pth"
model = torch.load(checkpoint_file)
model.eval()
```
on Pytorch == 0.4.1

I am getting error as:

``
key error: "track running status"
``

However on pytorch==0.3.1 the model loads fine.  Seems that the model is not compatible with pytorch version 0.4.1

**Comment by huster-wgm** [comment]
Yes, you are right. After updating to 0.4.1, I just save the params. I will update these checkpoints soon.
